__tests__/spec/utils.js

```diff
@@ -37,7 +37,8 @@ function getReleaseVersion () {
  * Return a path to the release archive for the current git index
  *
  * Creates a release archive from the git HEAD for the project we are currently
- * running tests in.
+ * running tests in. This will included uncommitted changes for tracked files, but
```

Comment:
I'm just being pedantic here, I think you mean "include uncommitted changes" instead of "included uncommitted changes".